src/main/kotlin/no/nav/familie/ef/sak/service/VurderingService.kt

```diff
@@ -75,6 +76,17 @@ class VurderingService(private val behandlingService: BehandlingService,
         return lagredeVilkårVurderinger + nyeVilkårVurderinger
     }
 
+    fun hentInngangsvilkårSomManglerVurdering(behandlingId: UUID): List<VilkårType> {
+        val lagredeVilkårVurderinger = vilkårVurderingRepository.findByBehandlingId(behandlingId)
+        val inngangsvilkår = VilkårType.hentInngangsvilkår()
+
+        return inngangsvilkår.filter {
+            lagredeVilkårVurderinger.find {
+                vurdering -> vurdering.type == it && vurdering.resultat == VilkårResultat.IKKE_VURDERT } !== null ||
+            lagredeVilkårVurderinger.none { vurdering -> vurdering.type == it }
+        }
```

Comment:
Jeg syns det er enklere hvis operator er etter radbryte, og så er det mulig å bytte ut `find !== null`
```suggestion
        return inngangsvilkår.filter {
            lagredeVilkårVurderinger.any { vurdering -> vurdering.type == it && vurdering.resultat == VilkårResultat.IKKE_VURDERT }
            || lagredeVilkårVurderinger.none { vurdering -> vurdering.type == it }
        }
```